app/src/main/AndroidManifest.xml

```diff
@@ -31,6 +31,7 @@
 
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
+                <action android:name="app.grapheneos.setupwizard.action.APPS" />
```

Comment:
as this doesn't reside inside suw, so  
name of this intent action should be `app.grapheneos.apps.MAIN`

Reply:
done